kyo-prelude/shared/src/main/scala/kyo2/internal/BaseKyoTest.scala

```diff
@@ -39,6 +39,15 @@ trait BaseKyoTest[S]:
     given eitherCanEqual[T, U]: CanEqual[Either[T, U], Either[T, U]] = CanEqual.derived
     given throwableCanEqual: CanEqual[Throwable, Throwable]          = CanEqual.derived
 
+    def retry[S](f: => Boolean < S): Boolean < S =
```

Comment:
I would probably rename this method `untilTrue` or `retryBoolean`, but doesn't matter too much as it's for testing.